spec/reverse_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './solver'
+
+describe 'the reverse of a string' do
+  before :each do
+    @solver = Solver.new
+  end
+
+  it 'should return the reverse of the given string' do
+    # fact = Solver.new.factorial(3)
```

Comment:
- [ ] Please delete commits that are not necessary to use.

Reply:
We didn't notice at first, thanks! :100: